.prow.yaml

```diff
@@ -1055,8 +1055,91 @@ presubmits:
             cpu: 2
           limits:
             memory: 6Gi
+            
+  #########################################################
+  # Konnectivity e2e tests
+  #########################################################
 
+  - name: pre-kubermatic-konnectivity-e2e
+    run_if_changed: "(.*[kK]onnectivity.*|pkg/resources/apiserver|.prow.yaml)"
+    decorate: true
+    clone_uri: "ssh://git@github.com/kubermatic/kubermatic.git"
+    labels:
+      preset-aws: "true"
+      preset-kubeconfig-ci: "true"
+      preset-docker-pull: "true"
+      preset-docker-push: "true"
+      preset-kind-volume-mounts: "true"
+      preset-vault: "true"
+      preset-goproxy: "true"
+    spec:
+      containers:
+        - image: quay.io/kubermatic/build:go-1.17-node-16-kind-0.11-5
+          imagePullPolicy: Always
+          command:
+            - "./hack/ci/run-konnectivity-e2e-test.sh"
+          env:
+            - name: VERSION_TO_TEST
+              value: v1.22.5
+            - name: KUBERMATIC_EDITION
+              value: ee
+            - name: SERVICE_ACCOUNT_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: e2e-ci
+                  key: serviceAccountSigningKey
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              memory: 4Gi
+              cpu: 2
+            limits:
+              memory: 6Gi
+  
   #########################################################
+  # Cilium e2e tests
+  #########################################################
+
+  - name: pre-kubermatic-cilium-e2e
+    # run_if_changed: "(.*(cilium|hubble|addon|connectivity-check|defaults|cni|validation|operator|webhook|crd).* | .prow.yaml)"
```

Comment:
uncomment?

Reply:
Failed to run the test with it. Will fix after debugging is done. Is there way to check this locally?